src/gourmand/backends/db.py

```diff
@@ -349,6 +350,26 @@ class Category(object):
 
         self._setup_object_for_table(self.categories_table, Category)
 
+    def setup_cuisine_table(self):
+        """Table to hold cuisine information.
+
+        Linked to Recipe table with recipe_id foreign key.  The value of
+        cuisine can be a country, like Indian or Thai, or a type of food
+        like Mom's Brunch Recipes or something like that.
+        """
+        self.cuisine_table = Table(
+            "cuisine",
+            self.metadata,
+            Column("id", Integer(), primary_key=True),
+            Column("recipe_id", Integer, ForeignKey("recipe.id"), **{}),
+            Column("cuisine", Text(), **{}),
+        )
+
+        class Cuisine(object):
```

Comment:
No need to subclass `object` in Python 3:

```suggestion
        class Cuisine:
```